app/presenters/api/v2/measures/measure_condition_component_presenter.rb

```diff
@@ -55,8 +55,7 @@ def duty_expression_str
         attr_reader :measure
 
         def apply_coerced_duty_amount_conversion_factor?
-          return false if Time.zone.today < excise_alcohol_coercian_starts_from
-          return false if MeasureCondition.point_in_time.present? && MeasureCondition.point_in_time < excise_alcohol_coercian_starts_from
+          return false if point_in_time.present? && point_in_time < excise_alcohol_coercian_starts_from
```

Comment:
We only need to care about the point in time not the actual current date otherwise these won't display until the 1st of August 2023